backend/canisters/community/api/src/queries/access_token.rs

```diff
@@ -0,0 +1,18 @@
+use candid::CandidType;
```

Comment:
This is only used for video call access tokens, right? If so the name should convey that.